packages/EasyHttpClient/tests/Bridge/EasyBugsnag/HttpRequestSentBreadcrumbListenerTest.php

```diff
@@ -0,0 +1,263 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyHttpClient\Tests\Bridge\EasyBugsnag;
+
+use Bugsnag\Breadcrumbs\Breadcrumb;
+use Bugsnag\Client;
+use Carbon\CarbonImmutable;
+use EonX\EasyHttpClient\Bridge\EasyBugsnag\HttpRequestSentBreadcrumbListener;
+use EonX\EasyHttpClient\Data\RequestData;
+use EonX\EasyHttpClient\Data\ResponseData;
+use EonX\EasyHttpClient\Events\HttpRequestSentEvent;
+use EonX\EasyHttpClient\Tests\AbstractTestCase;
+use Exception;
+use Mockery\MockInterface;
+
+class HttpRequestSentBreadcrumbListenerTest extends AbstractTestCase
+{
+    /**
+     * @param array<string, mixed> $expectedMetadata
+     *
+     * @dataProvider provideEvents
+     */
+    public function testItSucceeds(HttpRequestSentEvent $event, array $expectedMetadata): void
```

Comment:
Can we follow the current approach to naming?

Reply:
Done.